daprdocs/content/en/operations/hosting/self-hosted/self-hosted-with-docker.md

```diff
@@ -134,22 +135,19 @@ services:
   ... # Deploy other daprized services and components (i.e. Redis)
 
   placement:
-    image: "daprio/dapr"
+    image: "daprio/placement"
     command: ["./placement", "--port", "50006"]
     ports:
       - "50006:50006"
 
   scheduler:
-    image: "daprio/dapr"
-    command: ["./scheduler", "--port", "50007"]
+    image: "daprio/scheduler"
```

Comment:
@nmalocic - thanks for the PR 🎉 I see there are extra changes beyond the scope of this PRs intentions. Mind removing the extra files - you might be able to run `git restore` or similar. If the diff still shows, feel free to close and open a new PR if that is easiest.